waspc/data/Generator/templates/Dockerfile

```diff
@@ -47,6 +47,7 @@ WORKDIR /app
 COPY --from=server-builder /app/server/node_modules ./server/node_modules
 COPY server/ ./server/
 COPY db/ ./db/
+RUN cd server && npm run build
```

Comment:
I couldn't run this in `server-builder` since I need the source code, which is copied over here. I didn't want to fundamentally restructure this Dockerfile, and instead do the smallest change that will work, since we will rethink this process in the near future.

Reply:
I don't quite understand this!
So we need to build a TS project. Why not build it in server-builder? I guess you would need to copy some extra files then, but I guess that wouldn't be too hard? I don't think this is too bad either, it is probably a small impact, but I am curious.

Reply:
There is an added benefit if we did it in the `server-builder` step, we could copy only the `node_modules` and `dist` folder to the next step (avoiding copying over the `src` and everything else).

But we should still separate the copying of `src` and `package*.json` files to keep the layer caching benefits.

Reply:
Ok ok, you all convinced me, I will just refactor it a bit to do all the building where it probably should. The 1 line fix was just so appealing though :D